governance-exercises/VoteWizard_interpretation_exercise_01/README.md

```diff
@@ -0,0 +1,51 @@
+# VoteWizard Interpretation Exericse #1
+
+## Question #1: Do you think there is any ambiguity in these clauses? If so, where, and how would you rewrite the clauses to remove the ambiguity?
+
+### There are a few potential ambiguities
```

Comment:
One other fairly major one is how to judge what is an unintended consequence versus an intended consequence, versus a 'really obviously predictable but fairly negative' consequence.

Like, facilitators are empowered to use broad judgement when it related to unintended consequences or mistakes. The problem is determining whether the situation falls under this requires broad judgement in and of itself.